internal/integration/api/common.go

```diff
@@ -167,12 +167,12 @@ func (suite *CommonSuite) TestDNSResolver() {
 	suite.Require().Equal("", stdout)
 	suite.Require().Contains(stderr, "'index.html' saved")
 
-	stdout, stderr, err = suite.ExecuteCommandInPod(suite.ctx, namespace, pod, "nslookup really-long-record.dev.siderolabs.io")
+	stdout, stderr, err = suite.ExecuteCommandInPod(suite.ctx, namespace, pod, "dig really-long-record.dev.siderolabs.io")
```

Comment:
I thought it's not available by default in `alpine` container?

Reply:
@smira maybe can we install it using `apk add --update bind-tools` the final command will look like `apk add --update bind-tools && dig really-long-record.dev.siderolabs.io`

Reply:
Lets install it with `apk add --update bind-tools`.